flybywiresim/a32nx
# fix(gpws): "too low terrain" warning on 2nd departure

Fixes #2235

## Summary of Changes


`Mode4MaxRAAlt` wasn't reset after touchdown leading to "loo low terrain" warning after second departure.

## Screenshots (if necessary)



## References





## Additional context


![Max RA Alt](https://user-images.githubusercontent.com/39596827/108513194-8a090180-72c2-11eb-8e3f-21582ecb6f82.png)


Discord username (if different from GitHub): coolll.sk#0022

## Testing instructions


* after landing try departing again -> the "too low terrain" call out should not be heard (note: this change)
* try getting close to ground (airborne with GPWS sys on and flaps retracted) 
 * "too low terrain" should work nominally between 10 - 2450 AGL
 * try climbing out of it and check if the warning stops
 * try approach with incorrect flaps settings, warning should still be heard


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
